--- bakefile-0.1.1.20030930/src/writer.py.wiget	2003-09-29 14:16:48.000000000 +0200
+++ bakefile-0.1.1.20030930/src/writer.py	2003-09-30 12:22:49.000000000 +0200
@@ -174,10 +174,13 @@
     return (rulesdir, template)
 
 def invoke_em(writer, file, method):
-    import empy.em
+    try:
+      import em
+    except ImportError:
+     import empy import em
     rulesdir, template = __findWriter(writer)
     filename = tempfile.mktemp('bakefile')   
-    empy.em.invoke(['-I','mk',
+    em.invoke(['-I','mk',
                     '-I','writer',
                     '-I','utils',
                     '-I','os,os.path',
