--- bakefile-0.1.1.20031009/Makefile.am.wiget	2003-10-09 14:42:55.000000000 +0200
+++ bakefile-0.1.1.20031009/Makefile.am	2003-10-09 16:11:57.959522631 +0200
@@ -15,7 +15,7 @@
               output/autoconf-macros.empy \
               output/borland.empy \
               output/borland-opts.empy \
-              output/dmarg.empy \
+              output/dmars.empy \
               output/msvc.empy \
               output/msvc-opts.empy \
               output/msvc6prj.py \
